YeomanTemplates/generator-choAdmin-boilerplate/generators/app/templates/package.json

```
@@ -0,0 +1,72 @@
+{
+  "name": "@canadahelps/<%= packageName %>",
+  "version": "<%= apiVersion %>",
+  "description": "<description>",
+  "main": "src/index<.ts|.js|.tsx>",
+  "scripts": {
+    "prepublishOnly": "npm test",
+    "prepare": "run-s clean transpile",
+    "clean": "rimraf lib",
+    "copy-style": "cpx \"src/**/*.{css,scss}\" lib",
+    "transpile": "run-p transpile:babel transpile:declarations copy-style",
+    "transpile:babel": "babel src --out-dir lib --extensions \".ts,.tsx,.js,.jsx\"",
+    "transpile:declarations": "tsc --emitDeclarationOnly",
+    "build-storybook": "build-storybook",
+    "build:dev": "webpack --env.mode development",
+    "build": "webpack --env.mode production --display-error-details",
+    "deploy:dev": "npm run build:dev",
+    "deploy": "npm run build",
+    "storybook": "start-storybook -s ../../dist -p 9009",
+    "test:ci": "cross-env NODE_ICU_DATA=./node_modules/full-icu jest --ci",
+    "test:coverage": "cross-env NODE_ICU_DATA=./node_modules/full-icu jest --coverage",
+    "test:snapshots": "cross-env NODE_ICU_DATA=./node_modules/full-icu jest --updateSnapshot",
+    "test": "cross-env DEBUG_PRINT_LIMIT=30000 NODE_ICU_DATA=./node_modules/full-icu jest",
+    "watch:test": "cross-env NODE_ICU_DATA=./node_modules/full-icu jest --watch",
+    "watch": "webpack --env.mode development --watch"
```

please add additional watchers and a ci:all script
```suggestion
    "watch:test": "cross-env NODE_ICU_DATA=./node_modules/full-icu jest --watch",
    "watch:tsc": "tsc --noEmit --watch",
    "watch": "webpack --env.mode development --watch",
    "ci:all": "run-p test:ci lint:ci tsc:ci"
```